us.zoom.Zoom.json

```diff
@@ -49,6 +50,7 @@
                     "type": "script",
                     "dest-filename": "zoom.sh",
                     "commands": [
+                        "ln -s ~/.var/app/us.zoom.Zoom/config/zoomus.conf ~/.var/app/us.zoom.Zoom/.config",
```

Comment:
I'm tired, so just to make sure I understand it right: despite `--persist=.config`, Zoom looks for zoomus.conf elsewhere, ignoring XDG_SOMETHING var?

Reply:
Yes and no. Zoom looks in `~/.config`, but the problem is that the user still has to manually copy/paste or create a `zoomus.conf` because `--persist=.config` doesn't create it for you. The symlink is used to symlink `~/.var/app/us.zoom.Zoom/config/zoomus.conf to `~/.var/app/us.zoom.Zoom/.config`.

In case you still don't understand: the Zoom Flatpak looks inside `~/.var/app/us.zoom.Zoom/.config`, but for some reason `--persist=.config` still doesn't map `zoomus.conf` to `~/.var/app/us.zoom.Zoom/.config/zoomus.conf`, so the symlink is used to symlink that file to where the Zoom Flatpak looks.

But yes, Zoom doesn't follow XDG standards. It just hardcodes to the directories which is why the Zoom Flatpak is pain to get it to work.